core/src/main/scala/org/broadinstitute/dsde/rawls/entities/base/InputExpressionReassembler.scala

```diff
@@ -226,13 +229,15 @@ object InputExpressionReassembler {
        */
       val updatedEvaluatedRefMap: Try[JsValue] = inputOption match {
         case Some(input) => input.workflowInput.getValueType.getTypeName match {
-          case TypeNameEnum.OBJECT => inputExprWithEvaluatedRef.map(x => updateInputExprValueMapForObjectInput(x, input.workflowInput.getValueType.getObjectFieldTypes.asScala))
-          case _ => inputExprWithEvaluatedRef
+          case TypeNameEnum.OBJECT =>
+            val objectFields = Option(input.workflowInput.getValueType.getObjectFieldTypes).map(_.asScala).getOrElse(mutable.Seq.empty[ValueTypeObjectFieldTypes])
+            inputExprWithEvaluatedRef.map(x => updateInputExprValueMapForObjectInput(x, objectFields))
```

Comment:
for my edification: did some behavior of `asScala` change that necessitates this code update?

Reply:
I don't think it's `asScala`....it's more of `input.workflowInput.getValueType.getObjectFieldTypes` could be `null`, but we weren't doing the null check....It hasn't been failing existing tests, but while I was debugging some unrelated test failures, I notice this could've been `null`, so making this a bit more reliable